src/__tests__/lib/rules/prefer-in-document.js

```diff
@@ -66,13 +106,39 @@ const invalid = [
       `expect(${q}('foo')).not.toBeNull()`,
       `expect(${q}('foo')).toBeInTheDocument()`
     ),
+    invalidCase(
+      `expect(${q}('foo')) .not .toBeNull()`,
```

Comment:
@AntonNiklasson FYI i had to change around the fixer from what you had due to this use case here.  FYI using hard coded numbers in a fixer can be problematic for stuff like spacing etc.